### ISSUE TITLE
CMS, sitewide 508-defect-2 [SCREENREADER, KEYBOARD, COGNITION]: Links MUST look like links

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8253]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

```diff
! Team affected: VSA Public Websites; Project: Find Forms MVP
```

> **Please note:** This is an issue in the VSA benefit hubs CMS templates, and the responsibility of the front-end team.

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

> Previously documented in https://github.com/department-of-veterans-affairs/va.gov-team/issues/6507; also documented in https://github.com/department-of-veterans-affairs/va.gov-team/issues/8252; team is working on in https://github.com/department-of-veterans-affairs/va.gov-team/issues/8135

This is the same issue documented in https://github.com/department-of-veterans-affairs/va.gov-team/issues/6507.

The sentence text **MUST** not be included in the focusable link. The screen reader reads the heading and the paragraph all together in one continuous line, since the heading doesn't have (and shouldn't) punctuation. In JAWS, the sentences are read out as links, so for low-vision users there will be a cognitive disconnect because they don't see any visual hint of a link but hear a link.

![Screen Shot 2020-04-21 at 8 20 22 PM](https://user-images.githubusercontent.com/57469/79927377-8dff9500-840d-11ea-896c-310d9adb86b9.png)

## Point of Contact

**VFS Point of Contact:** Jennifer

## WCAG or Vendor Guidance (optional)

* [WebAIM Links and hypertext](https://webaim.org/techniques/hypertext/link_text)
* [WCAG 2.0 - Failure of Success Criterion 1.4.1 due to creating links that are not visually evident without color vision](https://www.w3.org/TR/WCAG20-TECHS/F73.html)
* [WCAG 2.0 AA SC 3.2.4 - Consistent Identification: Components that have the same functionality within a set of Web pages are identified consistently.](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-consistent-functionality.html)
* [WUGHCAG 3.2.4 - Plain language version of the above](https://www.wuhcag.com/consistent-identification/)
* [HTML5 Accessibility: aria-hidden and role=”presentation”](http://john.foliot.ca/aria-hidden/)

**Comment by jenstrickland** [comment]
@ncksllvn - is this indeed a sitewide issue? I think this may be a CMS issue.

**Comment by ncksllvn** [comment]
I've been thinking of those things as sort of the same thing because an issue with a CMS component will surface across many pages. But specifically I think this is a problem with the "list of link teasers" CMS template

**Comment by zacharymorel** [comment]
@jenstrickland this is ready for validation on https://staging.va.gov/find-forms/

**Comment by jenstrickland** [comment]
@zacharymorel **You're awesome!** Thank you for fixing this. Validated.
